src/Transmutable/Html.gren

```diff
@@ -121,11 +121,13 @@ is not covered by the helper functions in this library.
 node : String -> Array (Attribute msg) -> Array (Html msg) -> Html msg
 node tagName attributes children =
     Interior.Node
-        { key = Nothing
-        , tagName = tagName
+        { props = 
```

Comment:
Would probably call this `properties` to be consistent with the other keys in this object.

Reply:
Good call. Done.